en/setup/dev_env_linux_arch.md

```diff
@@ -1,27 +1,48 @@
-# Development Environment on ArchLinux
+# Development Environment on Arch Linux
 
-> **Note** These instructions allow you to build PX4 (without RTPS) for NuttX targets, using an unsupported version of GCCE from the package manager.
-The instructions have been tested on Antergos (an Arch Linux based distribution) as it is easier to set up than Arch Linux.
-We hope to provide fully tested instructions with the supported toolchain in the near future.
+> **Note** These instructions allow you to build PX4 (without RTPS) for NuttX targets and run simulation with jMAVsim or gazebo.
+The instructions have been tested on Manjaro (an Arch Linux based distribution) as it is much easier to set up than Arch Linux.
 
-## Permissions
+## Setup script
```

Comment:
@MaEtUgR Next time you do a docs push could you please push to the PX4 repo rather than a personal branch. That way I can apply technical editing in place.

Reply:
@hamishwillee I would love to, can you give me access? I have the same rights as any stranger here :smirk:

Reply:
@MaEtUgR welcome to the docs team.

Reply:
Thanks!